utils/config.mlp

```diff
@@ -204,6 +209,30 @@ let print_config oc =
   p "ast_intf_magic_number" ast_intf_magic_number;
   p "cmxs_magic_number" cmxs_magic_number;
   p "cmt_magic_number" cmt_magic_number;
+]
+
+let print_config_value oc = function
+  | String s ->
+      Printf.fprintf oc "%s" s
+  | Int n ->
+      Printf.fprintf oc "%d" n
+  | Bool p ->
+      Printf.fprintf oc "%B" p
 
+let print_config oc =
+  let print (name, valu) =
+    Printf.fprintf oc "%s: %a\n" name print_config_value valu in
+  List.iter print configuration_variables;
   flush oc;
 ;;
+
+let config_var name =
+  match List.assoc_opt name configuration_variables with
+  | None -> None
+  | Some v ->
+      let s = match v with
```

Comment:
It's a ~small~microscopic nit, but `valu`/`value` is used elsewhere to refer to a value, but `v` here